docs/management/admin/response-actions.asciidoc

```diff
@@ -111,6 +111,29 @@ TIP: You can use the <<use-osquery,Osquery manager integration>> to query a host
 When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a malware prevention alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`), which appears under *Highlighted fields* in the alert details flyout. Then paste the value into the `--path` parameter. 
 ====
 
+=== `execute`
+
+Run a shell command on the host. The command's output and any errors are returned in the response console, up to 2000 characters. The complete output (stdout and stderr) are also saved to a downloadable .zip archive (password: `elastic`). Use these parameters:
+
+* `--command` : (Required) A single-line command to run on the host. The command must be supported by the host operating system's default terminal shell (for example, bash for Linux, cmd.exe for Windows).
++
+[NOTE]
+====
+* Multiple consecutive dashes in the value must be escaped. For example, to represent a directory named `/opt/directory--name`, use the following: `/opt/directory\-\-name`.
```

Comment:
Interesting that dashes only need to be escaped if there are two, but in that case _both_ need to be escaped (not just the second one). Might be worth making explicit whether the user has to escape a single dash.

Reply:
If we're going to the trouble of mentioning that consecutive dashes require escaping, the reader would probably infer that single dashes _don't_ require it -- or else we would have mentioned them, too. (We do mention quote marks below, but that's kind of a special case, and it's a slippery slope to identify the characters that _don't_ require special handling.)

I tinkered with making this explicit anyway, but it started feeling a bit overwrought and distracting from the main point about the consecutive dashes.

Reply:
> I tinkered with making this explicit anyway, but it started feeling a bit overwrought and distracting from the main point about the consecutive dashes.

I agree here, I do think having an explicit callout about consecutive dashes is enough and users would understand.  At the most, maybe one sentence at the end would drive the point home - "Single dashes do not need to be escaped."

Reply:
```suggestion
* Multiple consecutive dashes in the value must be escaped; single dashes do not need to be escaped. For example, to represent a directory named `/opt/directory--name`, use the following: `/opt/directory\-\-name`.
```